File: src/main/Java/boj_step19/boj11050.java
import java.io.*;
import java.util.StringTokenizer;

public class boj11050 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
        int n = Integer.parseInt(tokenizer.nextToken());
        int k = Integer.parseInt(tokenizer.nextToken());
        int nfac = 1, kfac = 1;
        for (int i = n - k + 1; i <= n ; i++) nfac *= i;
        for (int i = 1; i <= k; i++) kfac *= i;
        writer.append(String.valueOf(nfac / kfac)).close();
    }
}

File: src/main/Kotlin/boj_step19/boj11050.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    var (n, k) = r.readLine().split(" ").map { it.toInt() }
    var nfact = 1
    var kfact = 1
    for (i in n - k + 1 .. n) nfact *= i
    for (i in 1 .. k) kfact *= i
    w.write("${nfact / kfact}")
    w.close()
}